py3status/modules/deadbeef.py

```diff
@@ -61,6 +71,12 @@ def post_config_hook(self):
         self.color_playing = self.py3.COLOR_PLAYING or self.py3.COLOR_GOOD
         self.color_stopped = self.py3.COLOR_STOPPED or self.py3.COLOR_BAD
 
+        # mix format and necessary placeholders with separator...
+        # then we merge together to run deadbeef command only once
+        self.placeholders = list(set(self.py3.get_placeholders_list(
+            self.format)) | set(FMT_PARAMETERS))
+        self.fmt = FMT_SEPARATOR.join(['%{}%'.format(x) for x in self.placeholders])
```

Comment:
you could just build `self.cmd` here and then it's all done in one place, and only done once